data/export.graphql

```diff
@@ -19,7 +19,6 @@ input ExportOptionsYolo {
 
 input ExportOptionsCsv {
   name: String
-  includeImageUrl: Boolean
 }
```

Comment:
Why 😿 ?

Reply:
As discussed on Monday's DSM, it should be by default, and it will also be by default on other export formats soon (unless with pick the option to include the whole images in the export)